Spark/metrics.csv

```diff
@@ -0,0 +1,207 @@
+spark.executor.total_duration.median,gauge,,millisecond,,The median total duration of the application's executors,0,spark,exe tot dur med
+spark.executor.total_duration.95percentile,gauge,,millisecond,,The 95th percentile total duration of the application's executors,0,spark,exe tot dur 95
+spark.executor.total_duration.count,rate,,millisecond,,The number of times the total duration of the application's executors was sampled,0,spark,exe tot dur cnt
+spark.executor.total_input_bytes.max,gauge,,byte,,The max total number of input bytes in the application's executors,0,spark,exe in byt mx
+spark.executor.total_input_bytes.avg,gauge,,byte,,The average total number of input bytes in the application's executors,0,spark,exe in byt avg
+spark.executor.total_input_bytes.median,gauge,,byte,,The median total number of input bytes in the application's executors,0,spark,exe in byt med
+spark.executor.total_input_bytes.95percentile,gauge,,byte,,The 95th percentile total number of input bytes in the application's executors,0,spark,exe in byt 95
+spark.executor.total_input_bytes.count,rate,,byte,,The number of times the total number of input bytes in the application's executors was sampled,0,spark,exe in byt cnt
+spark.executor.total_shuffle_read.max,gauge,,byte,,The max total number of bytes read during a shuffle in the application's executors,0,spark,exe tot shfl rd mx
+spark.executor.total_shuffle_read.avg,gauge,,byte,,The average total number of bytes read during a shuffle in the application's executors,0,spark,exe tot shfl rd avg
+spark.executor.total_shuffle_read.median,gauge,,byte,,The median total number of bytes read during a shuffle in the application's executors,0,spark,exe tot shfl rd med
+spark.executor.total_shuffle_read.95percentile,gauge,,byte,,The 95th percentile total number of bytes read during a shuffle in the application's executors,0,spark,exe tot shfl rd 95
+spark.executor.total_shuffle_read.count,rate,,byte,,The number of times the total number of bytes read during a shuffle in the application's executors was sampled,0,spark,exe tot shfl rd cnt
+spark.executor.total_shuffle_write.max,gauge,,byte,,The max total number of bytes written during a shuffle in the application's executors,0,spark,exe tot shfl wrt mx
+spark.executor.total_shuffle_write.avg,gauge,,byte,,The average total number of bytes written during a shuffle in the application's executors,0,spark,exe tot shfl wrt avg
+spark.executor.total_shuffle_write.median,gauge,,byte,,The median total number of bytes written during a shuffle in the application's executors,0,spark,exe tot shfl wrt med
+spark.executor.total_shuffle_write.95percentile,gauge,,byte,,The 95th percentile total number of bytes written during a shuffle in the application's executors,0,spark,exe tot shfl wrt 95
+spark.executor.total_shuffle_write.count,rate,,byte,,The number of times the total number of bytes written during a shuffle in the application's executors was sampled,0,spark,exe tot shfl wrt cnt

```

Comment:
a nitpick: I'd replace "number of bytes written during a shuffle" to "number of shuffled bytes" as this term is well established. @jeremykarn what do you think?